internal/installercache/installercache.go

```diff
@@ -247,6 +256,7 @@ func (i *Installers) shouldEvict(totalUsed int64) (shouldEvict bool) {
 //
 // Locking must be done outside evict() to avoid contentions.
 func (i *Installers) evict() bool {
+	i.metricsAPI.InstallerCacheTryEviction()
```

Comment:
Why do we want to register this metric differently than the other eviction metric? What is the idea behind it? Could you please explain a scenario where we'd need this info in this format (as opposed to not having this metric, compact it in the same metric, or possibly even switching metric type)?

Reply:
It is possible that you may attempt to evict and yet find no candidates. 

I wanted to ensure that we have a simple count of the number of times that eviction has been attempted.
This would be a good indicator of how often we end up with storage pressure.

Reply:
If there's storage pressure (for which IIRC we already have a metric for), we could check if there's any eviction going on (metric being worked on in this PR). If the eviction metric reports 0 evictions, something is fishy and we can check logs, where we could place a warning for the conditions you're worried about.

How would adding this metric help with this process?

Reply:
> If there's storage pressure (for which IIRC we already have a metric for)

Which metric is that?

Reply:
OK, let's remove this metric